apps/notifications/src/standalone/auth-wrapper/iframe-proxy.jsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import wpcom from 'wpcom';
+import proxyRequest from 'wpcom-proxy-request';
+
+/**
+ * Internal dependencies
+ */
+import setTracksUser from './set-tracks-user';
+
+const useIframeProxyClient = () => {
+	const [ client, setClient ] = useState( null );
+
+	useEffect( () => {
+		const tempClient = wpcom( proxyRequest );
+		tempClient.request( { metaAPI: { accessAllUsersBlogs: true } }, ( error ) => {
+			if ( error ) {
+				throw error;
+			}
+
+			setClient( tempClient );
+		} );
+	}, [] );
+
+	return client;
+};
+
+const IframeProxyWrapper = ( Wrapped ) => {
+	// This is a wrapped component, not a callback, so hooks are okay.
+	/* eslint-disable react-hooks/rules-of-hooks */
+	return ( { ...childProps } ) => {
```

Comment:
No need to destructure all fields like this. `( childProps ) => { ... }` does the same thing.